girder/girder
# Update CircleCI primary Docker image

This updates CircleCI to use an auto-built base image, to add deployment of master, and to test against the latest Python 3.5. This also fixes several Girder bugs with Python 3.5.

* Use the auto-built Docker ["girder/girder_test" image](https://hub.docker.com/r/girder/girder_test/) for CI
* Add a workflow step to deploy master
* Test Python 3.5 as the latest version
* Make MockSmtpReceiver compatible with Python 3.5
* Use PyMongo >3.5 to ensure support for latest MongoDB servers
* `json.loads` returns a `JSONDecodeError` in Python 3.5
